phoenix-core-client/src/main/java/org/apache/phoenix/coprocessorclient/BaseScannerRegionObserverConstants.java

```diff
@@ -62,6 +62,8 @@ public static long getMaxLookbackInMillis(Configuration conf){
     public static final String UPSERT_SELECT_EXPRS = "_UpsertSelectExprs";
     public static final String DELETE_CQ = "_DeleteCQ";
     public static final String DELETE_CF = "_DeleteCF";
+    public static final String UPSERT_STATUS_CQ = "_UpsertStatusCQ";
```

Comment:
We need to use this column family and qualifier when generating the Result object on the server and when reading from the client.